mockserver-api/src/main/xsd/pl/touk/mockserver/api/common.xsd

```diff
@@ -11,6 +11,7 @@
             <xs:enumeration value="HEAD"/>
             <xs:enumeration value="OPTIONS"/>
             <xs:enumeration value="PATCH"/>
+            <xs:enumeration value="ANY" />
```

Comment:
HTTP does not change very often, but maybe we should name it in safe way to be sure that it won't clash with new things e.g. `_ANY`?

Reply:
Done.